Governance/tech/conventions.md

```diff
@@ -1,147 +1,295 @@
+**Recommendations**
+
+* The value standard should be used instead of gregorian in the calendar attribute.
+* The attribute leap_month should not appear unless the attribute leap_year is present.
+* The time coordinate should not cross the date 1582-10-15 when the default mixed Gregorian/Julian calendar is in use.
 ```
 
-```{dropdown} long_name 
-CF - highly recommended 
-the [long_name](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#long-name) attribute provides a description of the variable. Either long_name or standard_name are expected to be present, ideally they both are. Standard_names are designed tcribed only a physical quantity, as air_temperature, this is usually not sufficient on its own to describe fully a variable.
-Long_name provides a chance to give a more complete description of the variable, highlighting aspect that cannot be specified in other attributes.
-For example if the variable is adimensional: percentage, 0-1, etc then the official units is "1". You can then specify what kind of adimensional units in the long_name. 
+## Variable attributes
+
+The CF Conventions provides a complete [list of variable attributes](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#attribute-appendix) it covers.<br>
+Here we are covering only the ones we believe are more critical. It is also worth to point out that only a few attributes are `required` to maintain backwards compatibility with COARDS. However, most of the `highly recommended` ones are expected for the files to be considered CF compliant, whenever they are applicable.
+
+````{dropdown} **units** 
+**CF - required**<br>
+Units are essential for all variables representing dimensional quantities.
+This is especially true for dimensions as latitude, longitude, and time coordinates which the netCDF library identifies solely by the value of their units attribute. This identification uses the [UDUNITS package](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#UDUNITS).<br> 
+Vertical coordinates with units of pressure may also be identified by the units attribute. Other vertical coordinates must use the attribute positive which determines whether the direction of increasing coordinate value is up or down. <br>
+CF provides the optional attribute axis for a direct identification of coordinates that correspond to latitude, longitude, vertical, or time axes.<br>
+
+**Requirements**
+* The value of the units attribute is a string from the UDUNITS library
+* If the variable is adimensional: percentage, 0-1, etc then the official units is always "1".
```

Comment:
unit or units? I'm actually not sure, 'units' parses wrong for me but I can see why it might be right.

Reply:
Do you mean the attribute name? Yes it's definitely units, but I reworded the last sentence as 
> If the variable is adimensional: percentage, 0-1, etc then the official value for units is always "1"

Reply:
Yes that makes more sense when phrased that way, thanks!